databricks/koalas/config.py

```diff
@@ -34,6 +34,11 @@
     # For example, this value determines whether the repr() for a dataframe prints out fully or
     # just a truncated repr.
     "display.max_rows": 1000,  # TODO: None should support unlimited.
+
+    # compute.shortcut_limit sets the shortcut value by using collected pandas dataframe directly.
```

Comment:
'compute.shortcut_limit' sets the limit for a shortcut. It computes specified number of rows and use its schema.